CHANGES.md

```diff
@@ -52,6 +52,9 @@
 - Support `(binaries)` in `(env)` in dune-workspace files (#5560, fix #5555,
   @emillon)
 
+- Attach `bin/main.exe` to the `dune` package. This ensures that dune has the
+  right build-info when it is not bootstrapped. (#5653, fix #5554, @emillon)
```

Comment:
Our change log is for end users, and I don't think this change impacts them in any way since building dune without bootstrapping was never a supported thing.

Reply:
good point!